command/agent/http.go

```diff
@@ -87,6 +88,7 @@ func NewHTTPServers(agent *Agent, config *Config, logOutput io.Writer) ([]*HTTPS
 			logger:   log.New(logOutput, "", log.LstdFlags),
 			uiDir:    config.UiDir,
 			addr:     httpAddr.String(),
+			prefix:   "https",
```

Comment:
If using the `string`, maybe at least have a `const` property defined in some place? It would at least avoid a magic string.